curvefs/conf/metaserver.conf

```diff
@@ -1,9 +1,38 @@
 #
 # metaserver服务端口
 #
-metaserver.listen.addr=127.0.0.1:6701
+metaserver.listen.addr=10.187.0.92:16701
 
 #
 # metaserver 日志打印目录
 #
-metaserver.common.logDir=/tmp/curvefs/metaserver
\ No newline at end of file
+metaserver.common.logDir=/tmp/curvefs/metaserver
+
+global.ip=10.187.0.92
+global.port=16701
+
+## copyset
+copyset.election_timeout_ms=1000
+copyset.snapshot_interval_s=1800
+copyset.catchup_margin=1000
+copyset.data_uri=local://./0/copysets
+copyset.raft_log_uri=local://./0/copysets
+copyset.raft_meta_uri=local://./0/copysets
+copyset.raft_snapshot_uri=local://./0/copysets
```

Comment:
currently, i didn't invole the filepool.
first, format filepool will cause a very long time, second, i think each raft log size is very smaller than 4K bytes, because we only modify an inode/dentry and don't have any other data. but i remember filepool needs minimum 4K write granularity, so it can waste disk space.

BTW, we haven't do performance test,  when after do this, and confirm raft-log really is bottlenecks, we can involve filepool again.
another solution may be we can save raft-log into higher performace disk like nvme, and save snapshot to ssd/hdd.

Reply:
ok